libraries/AP_HAL_ChibiOS/hwdef/QioTekZealotF427/hwdef-bl.dat

```diff
@@ -0,0 +1,49 @@
+# hw definition file for processing by chibios_hwdef.py
+# for for QioTekZealotF427 hardware bootloader from Qio-tek.com 
+
+MCU STM32F4xx STM32F427xx
+
+APJ_BOARD_ID 1021
```

Comment:
You need to add a new board type to https://github.com/ArduPilot/ardupilot/blob/master/Tools/AP_Bootloader/board_types.txt